lib/roger_ui/partitions.ex

```diff
@@ -0,0 +1,40 @@
+defmodule RogerUI.Partitions do
+  @moduledoc """
+  Generate Partition list from Roger.Info.partitions()
+  """
+  @doc """
+  Takes a Keyword list that contains the nodes, status and partitions with queues, like this:
+
+   [
+    "server@127.0.0.1": %{
+      running: %{
+        "roger_test_partition_1" => %{
+          default: %{consumer_count: 1, max_workers: 10, message_count: 740, paused: false},
+          fast: %{consumer_count: 1, max_workers: 10, message_count: 740, paused: false},
+          other: %{consumer_count: 1, max_workers: 2, message_count: 0, paused: false}
+        }
+      },...
+    ]
+
+  and transforms it into a partition list
+  """
+  def normalize(partitions) do
+    partitions
+    |> Enum.flat_map(&capture_node_name(&1))
```

Comment:
`&capture_node_name(&1)` is redundant and can be written as `&capture_node_name/1`

Reply:
done